src/util/avro.js

```diff
@@ -1,5 +1,27 @@
+const log = require('./logger').default;
 const avro = require('avsc');
 
+const getPlatformAnalyticsLog = (isGAE = process.env.GAE_INSTANCE) => {
```

Comment:
we could translate the env var into a config var in a separate file, but this seems like a clear case of an environment dependency as opposed to a 'local configuration' dependency - the environment itself must be able to support the automatic authentication necessary to make pubsub calls, and the env var seems like a good way of indicating that.